tools/toolchains/xcode_configure/xcode_configure.bzl

```diff
@@ -73,6 +75,12 @@ def _xcode_version_output(repository_ctx, name, version, aliases, developer_dir)
     tvos_sdk_version = _search_sdk_output(xcodebuild_result.stdout, "appletvos")
     macos_sdk_version = _search_sdk_output(xcodebuild_result.stdout, "macosx")
     watchos_sdk_version = _search_sdk_output(xcodebuild_result.stdout, "watchos")
+    versions_dict = {
```

Comment:
Pretty sure it'll wind up the same at the end; it'll just fail if it can't find the right setting (since I'm not using `get`). Willing to revisit this though.

Reply:
Cool, yeah I think the dict might make more sense: if we end up with dynamic access patterns!